### DISCUSSION TITLE
Is it possible to chain HTTP request?

### DISCUSSION THREAD

**Discussion by YannGarcia** [mockoon/mockoon#594] [Help] [answered]
Hello All,

First of all, many thanks for this nice tool.

I want to simulate a subscription mechanism: an application sends an HTTP POST to subscribe, receive one or more POST as notification and the application sends an HTTP DELETE to remove the subscription. The context is 3GPP T8 interface.
In practice, my application sends an HTTP POST to Mockoon, Mockoon sends the response. And now, I would like that Mockoon sends an HTTP POST after sending the HTTP response. Is it feasible?

Many thanks for your support,

BR\Yann

**Comment by 255kb** [comment, marked as answer]
Hi @YannGarcia thank you for the kind words 😃 
I am afraid you cannot achieve what you want with Mockoon. It has been designed to respond to entering requests only but not to make calls. So, while there are some rules that you can use, they will only determine which response is served. They cannot trigger any subsequent call.

**Comment by YannGarcia** [comment]
Hello Guillaume,

Thanks a lot for the information.

BR\Yann
